### ISSUE TITLE
Publish runtime to crates.io

### ISSUE THREAD

**Issue by FredrikNoren** [AmbientOrg/Ambient#41]
In order to publish to crates.io we need to get rid of the git dependencies:

- [ ] Profiling https://github.com/AmbientOrg/Ambient/blob/main/Cargo.toml#L19
- [ ] cap-std etc. https://github.com/AmbientOrg/Ambient/blob/main/Cargo.toml#L139
- [x] erased-serde https://github.com/AmbientOrg/Ambient/blob/main/crates/ecs/Cargo.toml#L14
- [x] russimp https://github.com/AmbientOrg/Ambient/blob/main/crates/model_import/Cargo.toml#L43
- [ ] wit-bindgen-host-wasmtime-rust https://github.com/AmbientOrg/Ambient/blob/main/crates/scripting/Cargo.toml#L44
- [x] physx-sys https://github.com/AmbientOrg/Ambient/blob/main/libs/physxx/Cargo.toml#L10

**Comment by philpax** [comment]
Quick update:
- I've opened [a PR](https://www.github.com/aclysma/profiling/pull/45) for profiling - once that's in, we can use the crates.io version again.
- cap-std is dependent on #1 
- wit-bindgen is dependent on #1 - they don't look ready to release, but I think we can probably set up a pre-release script that just instantiates the host code, similar to what we do for the guest
- physx-sys was updated yesterday 🎉. Just need them to cut a release.

Not too far off now!
